.changeset/wicked-clouds-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cmpsr/components": minor
+---
+
+reuse textstyles
```

Comment:
why do we have two changeset for this PR?, and IMO non of them actually reflects the change that is fixing the text style

Reply:
Updated